Makefile.uk

```diff
@@ -1,3 +1,3 @@
-$(eval $(call addlib,apphelloworldgo))
+$(eval $(call addgoapp,apphelloworldgo))
```

Comment:
Why do you have this change here?

Reply:
Because `libgo` has custom build mechanics, given by [this PR from Unikraft core](https://github.com/unikraft/unikraft/pull/1005/commits/84ecf07cb866fdb12b9ebd6529bb5d5a11fe9e67) and [this rule from `libgo`](https://github.com/unikraft/lib-libgo/blob/79a26b78464635325656dbadf08ef3ff26e30791/Makefile.rules#L164)